ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/impl/RowImplTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.google.common.collect.ImmutableList;
+import io.vertx.core.json.JsonArray;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RowImplTest {
```

Comment:
Again, not a fan of fine grained unit tests. I'd prefer to see the behaviour of a row tested on instance of Row _interface_ returned from the actual API rather than the particular implementation RowImpl. If we later change the implementation these kinds of tests get very brittle and hard to refactor whereas tests that test against the interface don't.

Reply:
As above.